ArduinoCode/Dashboard/Dashboard.ino

```diff
@@ -3,6 +3,16 @@
 
 #include <mcp_can.h> // uses seeed-studio's CAN_BUS_Shield library
 
+
+#define CAN_FAULT_LED      0
+#define GENERAL_LED_2      1 // custom behavior 
```

Comment:
Is LED2 = 1 and LED1 = 2 based on schematics? Cause for the sake of satisfaction we can definitely switch it